avm/res/app/container-app/main.bicep

```diff
@@ -91,7 +91,7 @@ param dapr object = {}
 param maxInactiveRevisions int = 0
 
 @description('Required. List of container definitions for the Container App.')
-param containers array
+param containers container[]
```

Comment:
Please note that changes such as this require an update of the module's readme. Fortunately, you don't have to do this yourself, but just need to run a script on the module's folder.

The script would be the `/utilities/tools/Set-AVMModule.ps1` script. Please dot-source it once and run it for the module folder as described [here](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/generate-bicep-module-files/#how-to-use-it). The script will essentially regenerate the `main.json` & `readme.md` file(s).

Reply:
![image](https://github.com/Azure/bicep-registry-modules/assets/169037533/454b27b6-877f-4c13-9dfa-1afdf281d5f8)
I ran it manually but it doesn't change anything to the readme or the main.json

Reply:
The changes should already be in the readme and the main.json

Reply:
In that case there seems to be something wrong with the utility. Will track this seperately